script.js

```diff
@@ -0,0 +1,480 @@
+      });
+    });
+  },
+  createKeysAndListners() {
+    this.elements.keysContainer.appendChild(this.createKeys());
+    this.keyDownAndUp();
+  },
+  keyDownAndUp() {
+    const keyboardKeys = document.querySelectorAll('.keyboard_key');
+    const keyCodesEng = {
+      Backquote: '`',
+      Digit1: '1',
+      Digit2: '2',
+      Digit3: '3',
+      Digit4: '4',
+      Digit5: '5',
+      Digit6: '6',
+      Digit7: '7',
+      Digit8: '8',
+      Digit9: '9',
+      Digit0: '0',
+      Minus: '-',
+      Equal: '=',
+      Backspace: 'Backspace',
+      Tab: 'Tab',
+      KeyQ: 'q',
+      KeyW: 'w',
+      KeyE: 'e',
+      KeyR: 'r',
+      KeyT: 't',
+      KeyY: 'y',
+      KeyU: 'u',
+      KeyI: 'i',
+      KeyO: 'o',
+      KeyP: 'p',
+      BracketLeft: '[',
+      BracketRight: ']',
+      Enter: 'Enter',
+      CapsLock: 'CapsLock',
+      KeyA: 'a',
+      KeyS: 's',
+      KeyD: 'd',
+      KeyF: 'f',
+      KeyG: 'g',
+      KeyH: 'h',
+      KeyJ: 'j',
+      KeyK: 'k',
+      KeyL: 'l',
+      Semicolon: ';',
+      Quote: "'",
+      Backslash: '\\',
+      ShiftLeft: 'ShiftL',
+      KeyZ: 'z',
+      KeyX: 'x',
+      KeyC: 'c',
+      KeyV: 'v',
+      KeyB: 'b',
+      KeyN: 'n',
+      KeyM: 'm',
+      Comma: ',',
+      Period: '.',
+      Slash: '/',
+      ShiftRight: 'ShiftR',
+      ControlLeft: 'CtrlL',
+      MetaLeft: 'Win',
+      AltLeft: 'AltL',
+      Space: '',
+      AltRight: 'AltR',
+      ArrowLeft: '◄',
+      ArrowUp: '▲',
+      ArrowDown: '▼',
+      ArrowRight: '►',
+      ControlRight: 'CtrlR',
+    };
+    const keyCodesRu = {
+      Backquote: '`',
+      Digit1: '1',
+      Digit2: '2',
+      Digit3: '3',
+      Digit4: '4',
+      Digit5: '5',
+      Digit6: '6',
+      Digit7: '7',
+      Digit8: '8',
+      Digit9: '9',
+      Digit0: '0',
+      Minus: '-',
+      Equal: '=',
+      Backspace: 'Backspace',
+      Tab: 'Tab',
+      KeyQ: 'й',
+      KeyW: 'ц',
+      KeyE: 'у',
+      KeyR: 'к',
+      KeyT: 'е',
+      KeyY: 'н',
+      KeyU: 'г',
+      KeyI: 'ш',
+      KeyO: 'щ',
+      KeyP: 'з',
+      BracketLeft: '[',
+      BracketRight: ']',
+      Enter: 'Enter',
+      CapsLock: 'CapsLock',
+      KeyA: 'ф',
+      KeyS: 'ы',
+      KeyD: 'в',
+      KeyF: 'а',
+      KeyG: 'п',
+      KeyH: 'р',
+      KeyJ: 'о',
+      KeyK: 'л',
+      KeyL: 'д',
+      Semicolon: ';',
+      Quote: "'",
+      Backslash: '\\',
+      ShiftLeft: 'ShiftL',
+      KeyZ: 'я',
+      KeyX: 'ч',
+      KeyC: 'с',
+      KeyV: 'м',
+      KeyB: 'и',
+      KeyN: 'т',
+      KeyM: 'ь',
+      Comma: ',',
+      Period: '.',
+      Slash: '/',
+      ShiftRight: 'ShiftR',
+      ControlLeft: 'CtrlL',
+      MetaLeft: 'Win',
+      AltLeft: 'AltL',
+      Space: '',
+      AltRight: 'AltR',
+      ArrowLeft: '◄',
+      ArrowUp: '▲',
+      ArrowDown: '▼',
+      ArrowRight: '►',
+      ControlRight: 'CtrlR',
+    };
+    document.addEventListener('keydown', (event) => {
```

Comment:
You can add your keybord listners once you created the keybord and not to add them any time you change the laguage of the keyboard. 
The problem here is that you add new EventListners but not delete the old ones so the amount of event listneres grows and store in computer memory, its main memory leak in JS. **Read about removing event lisners**